src/pattern-library/components/patterns/ColorsPage.tsx

```diff
@@ -5,12 +5,22 @@ import Library from '../Library';
 type ColorSwatchProps = {
   colorClass: string;
   colorName: string;
+  span?: 'third' | 'half';
 };
 
-function ColorSwatch({ colorClass, colorName }: ColorSwatchProps) {
+function ColorSwatch({
+  colorClass,
+  colorName,
+  span = 'third',
+}: ColorSwatchProps) {
   return (
     <div>
-      <div className={classnames(colorClass, 'w-64 h-8 mr-4')} />
+      <div
+        className={classnames(colorClass, 'h-8', {
+          'md:w-64': span === 'third',
+          'md:w-96': span === 'half',
+        })}
+      />
```

Comment:
There's room for improvement in this page layout, but I decided to keep things simple for now.

We can combe back end define a proper responsive grid later.